cmake/llvm-version.txt

```diff
@@ -1 +1 @@
-89946bda5e1c7ceaf6d26634cc8c8c9498d9f7be
+fbee8d5a65defae402f98794d778fe724d24cc51
```

Comment:
Shall we update llvm in a separate PR?

Reply:
Yes, I have reverted this change and will use a new PR if needed